### DISCUSSION TITLE
[Extreme Noob] Can't connect to my Friends external Jellyfin Server

### DISCUSSION THREAD

**Discussion by PalpoKelpo** [jellyfin/jellyfin-desktop#739] [General]
Hello Everyone!

So I've got the following problem:

I've been enjoying a show on my friend's Jellyfin server without any issues for about a week. Whether I was using the web client or the Jellyfin Media Player, everything worked perfectly.

However, starting yesterday, I’ve been experiencing a frustrating problem. Every time I try to log into my friend's external Jellyfin server, I get a connection failure. This happens on both the web client and the Jellyfin Media Player. Occasionally, it works for about 10 minutes, but then it fails again, leaving me unable to connect for hours at a time.

I'm using Tailscale to connect to the server. Does anyone have any ideas on why this might be happening and how to fix it?

Here are a few details that might help diagnose the issue:

    My setup: Windows 11 / MacOS - Using Tailscale to connect to the external Jellyfin server.

    Devices: Tried on multiple devices (Windows PC, iPad, iPhone) and experienced the same issue.

    Timing: Worked perfectly for a week before this issue started yesterday.

    Symptoms: Intermittent connectivity – sometimes works for a brief period, then fails repeatedly for hours.

Any insights or troubleshooting tips would be greatly appreciated!

Thanks in advance for your help!

Edit: And I just wanted to mention, that other friends that access his server don't have the same issue as me, so I believe the problem lies with me!

**Comment by thornbill** [comment]
See https://jellyfin.org/docs/general/getting-help on the correct process for getting support.
